acai-ios/CollectionViewCells/RadioSelectionCollectionViewCell.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  RadioSelectionCollectionViewCell.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 3/17/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+
+class RadioSelectionCollectionViewCell: UICollectionViewCell {
+    
+    // MARK: View vars
+    
+    var titleLabel: UILabel!
+    var priceLabel: UILabel!
+    var imageView: UIImageView!
+    var radioCircle: UIView!
+    var radioFill: UIView!
+    var line: UIView!
+    
+    // MARK: Data
+    
+    var customizationOption: CustomizationOption!
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        imageView = UIImageView()
+        imageView.image = UIImage()
+        imageView.contentMode = .scaleAspectFit
+        self.addSubview(imageView)
+        imageView.snp.makeConstraints { (make) in
+            make.leading.equalToSuperview().offset(24)
+            make.height.width.equalTo(55)
+            make.centerY.equalToSuperview()
+        }
+        titleLabel = UILabel()
+        titleLabel.text = ""
+        titleLabel.font = Acai.avenirNextBold.withSize(17)
+        titleLabel.textColor = .acaiBlack
+        self.addSubview(titleLabel)
+        titleLabel.snp.makeConstraints { (make) in
+            make.leading.equalTo(imageView.snp.trailing).offset(24)
+            make.bottom.equalTo(self.snp.centerY)
+            make.trailing.equalTo(titleLabel.snp.trailing)
+        }
+        priceLabel = UILabel()
+        priceLabel.text = ""
+        priceLabel.font = Acai.avenirNextMedium.withSize(14)
+        priceLabel.textColor = .acaiBlack
+        self.addSubview(priceLabel)
+        priceLabel.snp.makeConstraints { (make) in
+            make.leading.equalTo(titleLabel.snp.leading)
+            make.top.equalTo(self.snp.centerY)
+            make.trailing.equalTo(priceLabel.snp.trailing)
+        }
+        radioCircle = UIView()
+        radioCircle.backgroundColor = .white
+        radioCircle.layer.borderColor = UIColor.acaiColdGray.cgColor
+        radioCircle.layer.borderWidth = 2
+        radioCircle.layer.cornerRadius = 12
+        self.addSubview(radioCircle)
+        radioCircle.snp.makeConstraints { (make) in
+            make.trailing.equalToSuperview().offset(-30)
+            make.centerY.equalToSuperview()
+            make.height.width.equalTo(24)
+        }
+        radioFill = UIView()
+        radioFill.backgroundColor = .acaiOrange
+        radioFill.layer.cornerRadius = 5
+        self.addSubview(radioFill)
+        radioFill.snp.makeConstraints { (make) in
+            make.centerX.equalTo(radioCircle.snp.centerX)
+            make.centerY.equalToSuperview()
+            make.height.width.equalTo(10)
```

Comment:
move layout constants into file constants, cc @kevinchan159

Reply:
i would just make it a constant at the top of the class

Reply:
Actually I would prefer it into file constants and not a class property